### DISCUSSION TITLE
Fan profile hotkey

### DISCUSSION THREAD

**Discussion by bigggggert** [seerge/g-helper#3322] [General]
How would I change the hotkey for changing the fan profile from ctrl+shift+f5 to fn+f5?

**Comment by seerge** [comment]
@bigggggert both Ctrl+Shift+f5 and FN+f5 change profiles. First one is designed for external keyboards.

**Reply by bigggggert** [reply]
Well fm+f5 doesnt work for me. In logs it just says 
10/29/2024 9:17:18 AM: WMI event 153
10/29/2024 9:17:18 AM: Key: 153
But when I press ctrl+shift+f5 it says 
10/29/2024 9:17:19 AM: F5 Control, Shift
10/29/2024 9:17:19 AM: Mode = 0 : 0
10/29/2024 9:17:19 AM: VivoMode = 0 : 0
10/29/2024 9:17:19 AM: Power Mode 961cc777-2547-4f9d-8174-7d86181b8a7a -> 00000000-0000-0000-0000-000000000000:OK

**Comment by seerge** [comment]
@bigggggert what kind of model do you have ?  
```
10/29/2024 9:17:18 AM: WMI event 153
10/29/2024 9:17:18 AM: Key: 153
```

On all other models FN+F5 emits ``174``
https://github.com/seerge/g-helper/blob/a7e50f89ca0966ce8c1a4d05c7acba2b4143723b/app/Input/InputDispatcher.cs#L750C26-L750C29

**Reply by bigggggert** [reply]
wdym model? like computer model or model of something else. if you mean computer model its the GX501

**Reply by seerge** [reply]
@bigggggert try this build
[GHelper.zip](https://github.com/user-attachments/files/17560954/GHelper.zip)

**Reply by bigggggert** [reply]
That worked, thanks.
